docs/components/docs.jsx

```diff
@@ -17,7 +17,7 @@ import Sidebar from "./sidebar";
 class Docs extends React.Component {
 
   componentWillMount() {
-    ga.pageview("/victory/docs");
+    ga.pageview(`${routingConfig.base}/docs`);
```

Comment:
This is just a leftover change from the `projects.formidablelabs.com/victory` => `victory.formidable.com` move.